Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PARENT_VERSION=1.2.15-node16.14.0
+ARG PARENT_VERSION=1.2.19-node16.15.1
```

Comment:
should this change be done in a sep. ticket so we can monitor any potential impact as well as standardise across all other repos?

Reply:
Tests and pipeline should cover any impact.  I don't see how this is any different to patching any other dependency.

Major version updates I'd do separately.